src/onesignal.ts

```diff
@@ -37,3 +37,25 @@ export async function sendPushNotification(
   }
   await client.createNotification(push);
 }
+
+export async function sendReadingReminderPush(
+  userIds: string[],
+  at: Date,
+): Promise<void> {
+  if (!appId || !apiKey) return;
+
+  const push = new OneSignal.Notification();
+  push.app_id = appId;
+  push.include_external_user_ids = userIds;
+  push.send_after = at.toISOString();
+  push.contents = { en: 'Build a habit and become more knowledgeable' };
+  push.headings = { en: "Hi, It's reading time" };
+  push.url = addNotificationUtm(
+    process.env.COMMENTS_PREFIX,
+    'push',
+    'reminder',
+  );
+  push.chrome_web_badge =
+    'https://daily-now-res.cloudinary.com/image/upload/v1672745846/public/dailydev.png';
```

Comment:
I saw some notes on removing this image, wasn't possible?

Reply:
missed that. you're right

Reply:
notification looks crappy with that, so i'm keeping for now and letting design know